tests/test_ingest.py

```diff
@@ -56,6 +56,12 @@ def ingest_to_archives(self, dataset_name: str):
         self.runner.archived_run(dataset_fixture)
         return self.runner
 
+    def ingest_to_direct_archives(self, dataset_name: str):
```

Comment:
Does "direct" here mean "not via dsp"?
If so, I think it could be removed from the name.

Reply:
OK, we called it like this with Alexie to differentiate it from submission via DSP. I would leave it for now, while we still have the 2 different way for archiving, otherwise it would be hard to know which one are we executing.